### PR TITLE
[FrameworkBundle] add email_validation_mode option

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Comment by xabbuh** [comment]
This makes it possible to configure the default validation mode for the email validator according to the changes made in #24442.

**Comment by nicolas-grekas** [comment]
Cool. Then I propose to match the recipe for 4.1 and turn html5 on by default!

File: src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd [hunk 1]
```diff
@@ -199,6 +199,8 @@
         <xsd:attribute name="cache" type="xsd:string" />
         <xsd:attribute name="enable-annotations" type="xsd:boolean" />
         <xsd:attribute name="static-method" type="xsd:boolean" />
+        <xsd:attribute name="strict-email" type="xsd:boolean" />
```
Comment (xabbuh) [inline src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd]: This option must also be added in older versions (see #25480).

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**